user_one.rb

```diff
@@ -0,0 +1,21 @@
+module MyEnumerable
+  def all?
+    each { |item| return false unless yield(item) }
+    true
+  end
+end
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] It would be a good thing according to the project requirements to ensure that your initialize method is able to accept multiple/ unspecified arguments. This would be implemented something like this:

```
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

- [ ] Also, your @list instance variable is supposed to be assigned to an empty array as shown above. Thanks to that, calling the constructor with a list of numbers is possible eg. MyList.new(1, 2, 3, 4)